### DISCUSSION TITLE
Flux Error: Python Has Stopped Working

### DISCUSSION THREAD

**Discussion by kwalitykontrol1** [comfyanonymous/ComfyUI#4189] [Q&A]
I am trying to use Flux in Comfy UI. It was working when I first installed it and I can use any other model just fine, but suddenly every time I hit Queue Prompt for Flux, I get an error Python Has Stopped Working. I have restarted both Comfy and my PC multiple times and it keeps happening. Any suggestions? Thanks.

**Comment by ltdrdata** [comment]
Try monitoring the memory usage with task manager while you do it.

**Comment by BrechtCorbeel** [comment]
try updating python?

**Comment by sky958958** [comment]
I also encountered this situation and had to restart my computer to run Flux properly
